lib/open-search-client.test.ts

```diff
@@ -312,24 +312,15 @@ describe('open-search-client', () => {
   describe('createIndex()', () => {
     it('creates the requested index', async () => {
       const indexName = randomIndexName();
-
       await osClient.createIndex(esUrl, indexName);
 
       expect(await osClient.indexExists(esUrl, indexName)).toBe(true);
     });
-
-    it('throws an Error if the index already exists', async () => {
```

Comment:
This functionality changed but the test was being skipped because of a .only in this file. Same for this test in `createTermsIndex`.